coops-ui/src/components/AnnualReport/Certify.vue

```diff
@@ -61,7 +61,7 @@ export default class Certify extends Vue {
 
   /**
    * Computed value.
-   * @return The trimmed "Certified By" string (may be '').
+   * @returns The trimmed "Certified By" string (may be '').
```

Comment:
Just curious: VSCode doesn't like @return? PyCharm is fine with the synonym (it originated in JavaDoc). Good to be consistent and use the more common one, though :)

Reply:
Actually, VS Code allows anything that starts with @ :/
So I changed all of these to use the JSDoc tag (https://jsdoc.app/tags-returns.html).